app/components/courses/no_results_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="app-search-results">
+  <% if devolved_nation? %>
+    <h2 class="govuk-heading-m">This service is for courses in England</h2>
+  <% end %>
+
+  <% case country
+       when "Scotland" %>
```

Comment:
This might be the nittiest comment ever but is this indentation right? I don't remember seeing multiline ERB